cfpb/consumerfinance.gov
# Remove lightbox2 from kbyo compare page

We published the `compare` companion page to `know-before-you-owe` essentially as a blog-post-as-custom-template. This included the custom dependency `lightbox2` which is only used on this subpage and yet has its own gulp subtask supporting its inclusion.

This page has gotten about 300 visits in CY22, a high-percentage of which, I'd wager, are visits from crawlers.

Thus, I'm chopping out `lightbox2` and making that page a little less fancy to help focus our build infrastructure on the 99.994% of other page views.

Testing:
 - Pull branch
 - visit http://localhost:8000/know-before-you-owe/compare/
 - Note that the `png`s are still viewable, but don't have the lightbox treatment
